README.md

```diff
@@ -94,16 +104,17 @@ You are encouraged to also create your own library of small, reusable functions
       },
     };
 
-    const rules: IRules<IPerson> = {
-      job: (X) => X.job(),
+    const rules: IRules<IRecord, IPerson> = {
+      job: (X) => X.job(''),
       name: {
-        first: (X) => X.name().split(' ')[0],
-        last: (X) => X.name().split(' ')[1],
+        first: (X) => X.name('').split(' ')[0],
+        last: (X) => X.name('').split(' ')[1],
```

Comment:
You know this doesn't work for many names... "Alice de Temple", "Jose Villanueva Borgos". But as an example, it doesn't really matter.